packages/react/src/alert/index.test.tsx

```diff
@@ -41,13 +44,14 @@ describe('CdsAlert', () => {
         </CdsAlertGroup>
 
         <CdsAlertGroup type="banner">
-          <CdsAlert status="info">
+          <CdsAlert status="info" aria-describedby="id">
```

Comment:
I couldn't get around the autogenerated id, even by supplying my own describedby, so skipped for now